templates/taxi/manufacturer_list.html

```diff
@@ -7,7 +8,10 @@ <h1>
       Create
     </a>
   </h1>
-
+  <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="🔎">
```

Comment:
The submit button for the search form is styled with Bootstrap classes and includes an emoji for a search icon.